edge-modules/MetricsCollector/Settings.cs

```diff
@@ -77,8 +79,8 @@ public override string ToString()
         {
             var fields = new Dictionary<string, string>()
             {
-                { nameof(this.AzMonWorkspaceId), this.AzMonWorkspaceId },
-                { nameof(this.AzMonLogType), this.AzMonLogType },
+                { nameof(this.LogAnalyticsWorkspaceId), this.LogAnalyticsWorkspaceId },
```

Comment:
same for LogAnalyticsLogType 
```suggestion
                { nameof(this.LogAnalyticsWorkspaceId), this.LogAnalyticsWorkspaceId ?? string.Empty },
```